### DISCUSSION TITLE
Invalid or unexpected token

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5305] [Help]
Hi All,

I just create a new laravel project, and installing filament.
But when trying sign-in for first time,. it give me error : invalid or unexpected token.

![image](https://user-images.githubusercontent.com/43429317/208972829-b2e3609a-2915-4cce-a4f3-72f37115042b.png)

Please help. How to fix this ?
Thank you

**Comment by ArtMin96** [comment]
Are you using nginx?

**Reply by altimusin** [reply]
No.. 
I am using my development PC.
Before this, if I created a new project with filament  in same PC, is working fine.

**Reply by ArtMin96** [reply]
You can check this. [https://github.com/livewire/livewire/issues/2338](https://github.com/livewire/livewire/issues/2338)
